### PR TITLE
Update microsoft-remote-desktop from 10.6.6 to 10.6.7

### PR DESCRIPTION
**Pull request by eloyaldea**
Created with `brew bump-cask-pr`.

### CODE REVIEW

File: Casks/microsoft-remote-desktop.rb [hunk 1]
```diff
@@ -14,7 +14,6 @@
   end
 
   auto_updates true
-  depends_on macos: ">= :high_sierra"
```
Comment (cho-m) [inline Casks/microsoft-remote-desktop.rb:17]: @bevanjkay I think there is still a restriction, just not properly documented.

From extracted PKG:
- Distribution
    ```xml
        <allowed-os-versions>
            <os-version min="10.14"/>
        </allowed-os-versions>
    ```
- Resources/en.lproj/Localizable.strings
    ```
    "require-min-os-title" = "Software Requirement";
    "require-min-os-message" = "This software requires macOS version 10.14.0 or later.";
    "require-min-os-message-preview" = "This software requires macOS version 10.14.0 or later.";
    ```

Comment (bevanjkay) [reply Casks/microsoft-remote-desktop.rb]: Thanks @cho-m - and thanks for the suggestion to check requirements.

**Review by bevanjkay — APPROVED** [review]
Thank you @eloyaldea

**Comment by eloyaldea** [comment]
Happy to help 👍

**Pull request merged**